core/tests/webdriverio_utils/forms.js

```diff
@@ -53,18 +53,17 @@ var GraphEditor = function(graphInputContainer) {
       '.e2e-test-Add-Node-button');
     await action.click('Add Node Button', addNodeButton);
     // Offsetting from the graph container.
-    await browser.moveToElement(graphInputContainer, xOffset, yOffset);
-    await browser.positionClick();
+    await graphInputContainer.click({x: xOffset, y: yOffset});
   };
 
   var createEdge = async function(vertexIndex1, vertexIndex2) {
     var addEdgeButton = await graphInputContainer.$(
       '.e2e-test-Add-Edge-button');
     await action.click('Add Edge Button', addEdgeButton);
-    await browser.moveToElement(vertexElement(vertexIndex1));
-    await browser.buttonDown();
-    await browser.moveToElement(vertexElement(vertexIndex2));
-    await browser.buttonUp();
+    var vertexElement1 = await vertexElement(vertexIndex1);
+    var vertexElement2 = await vertexElement(vertexIndex2);
+
+    await vertexElement1.dragAndDrop(vertexElement2);
```

Comment:
We also do not have any replacement for buttonDown or buttonUp, here the action that we want to perform is move the cursor to `vertexElement1` and then left click and without releasing the button move the cursor to `vertexElement2` and then release the butto and in webdriverio we do the same with just one command `dragAndDrop` so I am using this. `await` is required at line 63 and 64 because without it dragAndDrop will not work (dragAndDrop works only with resolved promise and without using await the element selector returns unresolved promise)